odoo/addons/base/models/ir_actions_report.py

```diff
@@ -709,6 +716,7 @@ def barcode(self, barcode_type, value, **kwargs):
             if kwargs['quiet']:
                 kwargs['barBorder'] = 0
 
+        from odoo.tools.barcode import check_barcode_encoding  # noqa: PLC0415
```

Comment:
Why this one is lazy loaded ? :thinking:

Reply:
Ok, updated.